include/alpaka/vec/Vec.hpp

```diff
@@ -656,6 +656,25 @@ namespace alpaka
         return r;
     }
 
+    //! \brief Converts std::array to alpaka Vec
+    //! \tparam TVal is the element type of the array
+    //! \tparam N is the size of the array
+    template<typename TVal, size_t N>
+    ALPAKA_FN_HOST_ACC constexpr auto arrayToVec(std::array<TVal, N> const& a) -> Vec<DimInt<N>, TVal>
```

Comment:
There is no need for an additional method that can only handle std array.
We have the traits `GetExtents` to provide the translation that alpaka can work with alpaka foreign types.

Reply:
Ok I used getExtents. There is already one implemented for 1D array arguments in "ViewPlanePtr" therefore I had to check the dimension. I will refine the solution. Thanks a lot.